browser-switch/src/main/java/com/braintreepayments/browserswitch/BrowserSwitchRequest.java

```diff
@@ -7,34 +7,24 @@
 
 class BrowserSwitchRequest {
 
-    private Uri uri;
-    private String state;
+    final private Uri uri;
     final private int requestCode;
-    private JSONObject metadata;
-
-    static final String PENDING = "PENDING";
-    static final String SUCCESS = "SUCCESS";
+    private final JSONObject metadata;
```

Comment:
```suggestion
    final private JSONObject metadata;
```
Just to be consistent 🙂